storage/storage.go

```diff
@@ -0,0 +1,28 @@
+package storage
+
+import (
+	"io"
+	"time"
+)
+
+type Storage interface {
+	Get(key string) (io.ReadCloser, error)
+	Put(user, key string, bytes []byte) error
+	Delete(key string) error
+	List(prefix string) ([]*Object, error)
+}
+
+type Object struct {
+	Key          string
+	LastModified *time.Time
+	Storage      Storage
+}
+
+func NewStorageProvider(provider string) Storage {
```

Comment:
Is it possible to select different Casdoor providers to use for each Casibase store? A provider name will be specified in the store edit page